### ISSUE TITLE
islands apart

### ISSUE THREAD

**Issue by forresto** [cblgh/rotonde-mentions#1]
I think I'm in an island with nobody following me, so I don't think the scraper will find me?

dat://e2608cc1cf0ba62df6b563232d2c37f2fa29d1ef62c6176f8b8f13516d08a46f/

**Comment by cblgh** [comment]
Yeah that's one of the side-effects of the current system. I followed you!

(In the future the scraper can be extended to take rotonde.html into account to remedy that problem:)
